### ISSUE TITLE
Font preference applied on-the-fly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#429]
```
What steps will reproduce the problem?
1. Configure font preference in $HOME/.minttyrc (e.g. DejaVu)
2. Don't have font installed (e.g. after Windows reinstallation)
3. Run mintty
4. Install font

What is the expected output?
I'd expect the now available font is only used after starting mintty again.

What do you see instead?
Mintty uses the preferred font instantly, within the running terminal.
It does not change its metrics, however, i.e. spacing will look weird.

```

Original issue reported on code.google.com by `towom...@googlemail.com` on 3 Jun 2014 at 1:29

**Comment by mintty** [comment]
Likewise, if the window border geometry changes by Personalization (size of border/title bar/scrollbar) or the Performance option "Use visual styles on windows and borders" is changed, the window does not look good without a refresh.

**Comment by mintty** [comment]
Released in 2.2.3.
